x/consensus/keeper/keeper.go

```
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	cmtproto "github.com/cometbft/cometbft/api/cometbft/types/v1"
```

golangci-lint (1.62.2)

ST1019: package "github.com/cometbft/cometbft/api/cometbft/types/v1" is being imported more than once

(stylecheck)